tools/testrunner/testrunner.go

```diff
@@ -139,6 +129,11 @@ func (r *runner) newAttempt() *attempt {
 	a := &attempt{
 		runner: r,
 		number: len(r.attempts) + 1,
+		coverProfilePath: fmt.Sprintf(
+			"%v_%v%v",
+			strings.TrimSuffix(r.coverProfilePath, codeCoverageExtension),
+			len(r.attempts),
+			codeCoverageExtension),
```

Comment:
Without this, coverage reports would override each other. Since they are additive, we just put them next to each other in the output dir.